src/dsp_tools/commands/excel2xml/excel2xml_lib.py

```diff
@@ -52,22 +52,38 @@ def make_xsd_id_compatible(string: str) -> str:
         BaseError: if the input cannot be transformed to an xsd:ID
 
     Returns:
-        an xsd ID based on the input string
+        an xsd ID based on the input string with a UUID attached.
     """
+    res = xsd_id_remove_illegal_characters(string)
+    # add uuid
+    _uuid = uuid.uuid4()
+    res = f"{res}_{_uuid}"
+    return res
+
+
+def xsd_id_remove_illegal_characters(string: str) -> str:
+    """
```

Comment:
Some more background info is necessary:

```suggestion
    """
    Make a string compatible with the constraints of xsd:ID, so that it can be used as "id" attribute of a <resource>
    tag. An xsd:ID must not contain special characters, and it must be unique in the document.
```